Containers/talk/supervisord.conf

```diff
@@ -8,12 +8,19 @@ logfile_maxbytes=50MB
 logfile_backups=10                              
 loglevel=error
 
+[program:cron]
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+command=/cron.sh
```

Comment:
Honestly I dislike that we need to do this. Cannot we simply recommend people with a dynamic dns to just use the built-in backup solution which will restart their server once a day? We actually already have the same problem with collabora already...

Reply:
We do not offer this in our eturnal container either, because it is messy. So I agree with you. I hope most people (your "80%") do not run their TURN service with their home fritz.box xD

Reply:
so removing the dyndns function?

Reply:
Yes, remove the cron.sh again